packages/spelunker-web/package.json

```diff
@@ -18,18 +18,12 @@
     "wowser"
   ],
   "dependencies": {
-    "apollo-cache-inmemory": "^1.6.6",
-    "apollo-client": "^2.6.10",
-    "apollo-link": "^1.2.14",
-    "apollo-link-error": "^1.1.13",
-    "apollo-link-http": "^1.5.17",
+    "@apollo/client": "^3.4.17",
     "classnames": "^2.3.1",
     "graphql": "^14.6.0",
-    "graphql-tag": "^2.10.4",
```

Comment:
`gql` is available as an export from `@apollo/client`, instead of requiring us to pull in the dep ourselves.